docs/usage/administration/admin_api/background_updates.md

```diff
@@ -82,3 +81,37 @@ The API returns the `enabled` param.
 ```
 
 There is also a `GET` version which returns the `enabled` state.
+
+
+## Run
+
+This API runs specific background updates. The job starts immediately after calling the API.
+
+
+The API is:
+
+```
+POST /_synapse/admin/v1/background_updates/start_job
+```
+
+with the following body:
+
+```json
+{
+    "job_name": "populate_stats_process_rooms"
+}
+```
+
+The API returns the `job_name` param.
```

Comment:
Why return the `job_name`?

Reply:
There is no special reason. What is better? Any ideas?Return nothing? Only status `200`?

Reply:
I'm not sure, I think just a `200` would be fine? I see that the enable API returns something similar, but I don't see the point in returning an input parameter?

Reply:
Done